### PR TITLE
GCP test runners

### PR DESCRIPTION
**Pull request by andy-neuma**
SUMMARY:
* switch "remote push" to use GCP based runners
* update workflows from "neuralmagic" with "nm-" and update associated callsites
* in benchmarking, allow using AWS
* delete "install test whl" action in favor of separate actions, "install whl" and "test whl"
* run "download-images.sh" out side of `run-tests`

TEST PLAN:
runs on remote push

### CODE REVIEW

File: .github/scripts/run-tests [hunk 1]
```diff
@@ -99,7 +99,7 @@ done
 echo "..."
 
 # download required artifacts for testing
-(cd ${TEST_DIR} && bash ../.buildkite/download-images.sh)
+# (cd ${TEST_DIR} && sudo bash ../.buildkite/download-images.sh)
```
Comment (dhuangnm) [inline .github/scripts/run-tests:102]: Is this intended to comment it out?

File: .github/workflows/nm-benchmark.yml [hunk 2]
```diff
@@ -138,18 +147,22 @@ jobs:
 
       - name: mount EFS
         run: |
-            sudo mkdir -m 777 -p /EFS
-            sudo mount -t nfs4 -o nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2,noresvport ${{ secrets.HF_FS_CACHE }}:/ /EFS
-            sudo chown -R $(whoami):$(whoami) /EFS
+            if [[ ${{ inputs.label }} == *aws* ]]; then
```
Comment (dbarbuzzi) [inline .github/workflows/nm-benchmark.yml:150]: You could move this `if` to the step itself if we don't need that particular pattern matching; something like:
```
if: startsWith(inputs.label 'aws')
```

**Review by dbarbuzzi — APPROVED** [review]

**Comment by derekk-nm** [comment]
You might want to update the comment in neuralmagic/benchmarks/README.md pointing to where the run results are now going.

**Review by derekk-nm — APPROVED** [review]

File: .github/workflows/nm-benchmark.yml (hunk 2 above)
Comment (andy-neuma) [reply .github/workflows/nm-benchmark.yml:150]: i'll get it on the next iteration.

File: .github/scripts/run-tests (hunk 1 above)
Comment (andy-neuma) [reply .github/scripts/run-tests:102]: yes

**Pull request merged**